cmd/kubelet/app/server.go

```diff
@@ -649,13 +649,13 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies, stopCh <-chan
 	// If the kubelet config controller is available, and dynamic config is enabled, start the config and status sync loops
 	if utilfeature.DefaultFeatureGate.Enabled(features.DynamicKubeletConfig) && len(s.DynamicConfigDir.Value()) > 0 &&
 		kubeDeps.KubeletConfigController != nil && !standaloneMode && !s.RunOnce {
-		if err := kubeDeps.KubeletConfigController.StartSync(kubeDeps.KubeClient, kubeDeps.EventClient, string(nodeName)); err != nil {
+		if err := kubeDeps.KubeletConfigController.StartSync(kubeDeps.KubeTPClients[0], kubeDeps.EventClient, string(nodeName)); err != nil {
```

Comment:
kubelet config controller sounds like a component belonging to RP, instead of TP1. We may need to have a better understanding of this component first.

Reply:
this will need further discussion on design. Will be fixed later.

Reply:
OK. Please consider it resolved.